src/site/content/en/blog/badging-api/index.md

```diff
@@ -87,37 +84,12 @@ Examples of sites that may use this API include:
 
 ## How to use the Badging API {: #use }
 
-The Badging API is available as an origin trial in Chrome 79 and later.
-
 The Badging API works on Windows, and macOS. Support for Chrome OS is in
 development and will be available in a future release of Chrome.
 On Android, the Badging API is not supported. Instead, Android automatically
-shows a badge on a web app when there is an unread notification, just like
+shows a badge on a web app when there is an unread notification, just as
```

Comment:
"just as for native apps" reads oddly to me, would "the same as native apps" be better?